atari/800xl/lib/chrout.c

```diff
@@ -1,12 +1,14 @@
 void __chrout(char c) {
```

Comment:
Instead, just "__attribute__((noinline)) void __chrout(char c) {"

This saves the extra overhead and is clearer about the intent.

Reply:
done, works like a charm!